### ISSUE TITLE
schedule/rooms: timezone mismatch

### ISSUE THREAD

**Issue by q9f** [pretalx/pretalx#1468]
### Problem and impact

When trying to set a time slot for a room in the room editor, and a custom timezone is set (here: Berlin/Europe), this timezone seems not to be reflected in the schedule editor, showing a different timezone (UTC? Server Time?).

### Expected behaviour

The room editor and the schedule editor should adhere to the same timezone settings.

### Steps to reproduce

1. Create (or edit) a room and add a timeslot, e.g., 9am - 9pm (respecting the timezone setting, here: Berlin/Europe)
2. Save the room
3. Go to schedule editor
4. See the room shows 8am - 8pm instead of what we have set above

### Screenshots

This is hard to screenshot because it is on two different pages but you should be able to reproduce this.

@ligi is administering the instance and might have more details about server and configuration.

### Link

https://speak.ticketh.xyz/orga/event/protocol-berg/schedule/

**Comment by rixx** [comment]
Duplicate of #1444 and related issues. Fixed on main, next release is waiting on the merge of #1340

**Comment by ligi** [comment]
thanks for the info @rixx - should we wait for the next release or run main for now?

**Comment by rixx** [comment]
I wish I could promise a release timeline (it's quite urgent in my mind, but  then, it has been quite urgent for over half a year now!). I'd recommend you switch to `main` in the meantime, which is also what pretalx.com is running. 

A good stable point before a bunch of new changes would be 678b4682 – you should be able to install that commit directly with

> pip3 install --user --upgrade-strategy eager -U "git+https://github.com/pretalx/pretalx.git@678b4682#egg=pretalx&subdirectory=src"

(or something equivalent, depending on your setup). Really sorry for the hassle – I had hoped to work on the last blocker during the Easter holidays, but then my laptop died unceremoniously, and I've been catching up on work and things to do ever since.
